sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsClient.java

```diff
@@ -53,7 +54,7 @@ public DigitalTwinsServiceVersion getServiceVersion() {
     // this annotation lets users know this method makes a call to the service and whether it returns a single resource or a collection of resources
     @ServiceMethod(returns = ReturnType.SINGLE)
     // TODO This is just a temporary implementation for test purposes. This should be spruced up/replaced once this API is actually designed
-    public DigitalTwinsGetByIdResponse getDigitalTwin(String digitalTwinId) {
+    public Response<Object> getDigitalTwin(String digitalTwinId) {
```

Comment:
This works in the sync client, but not in the async client. We'll still need the java autorest team to fix their bug so that the protocol layer actually returns the expected type

Reply:
I may have missed it but what and why does it not work for async?